moe.karaokes.mugen.app.yml

```diff
@@ -0,0 +1,93 @@
+app-id: moe.karaokes.mugen.app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: km-wrapper
+separate-locales: false
+finish-args:
+  - '--filesystem=host'
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
```

Comment:
does it support wayland?

Reply:
Now it does, I just had it tested by a friend. Thanks for letting us know it could be an issue.